op-chain-ops/script/console.go

```diff
@@ -0,0 +1,22 @@
+package script
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type ConsolePrecompile struct {
+	logger log.Logger
+	sender common.Address
+}
+
+func (c *ConsolePrecompile) log(ctx ...any) {
+	// frame := c.h.CurrentCall()
```

Comment:
Did you mean to leave in these comments?

Reply:
Yes, it's being swapped in the forge-script PR, for now it's just using the `ConsolePrecompile` local state, to uncouple the PR.